### PR TITLE
Desktop version (Complete)

### PR DESCRIPTION
**Pull request by lily4178993**
In this branch, I have built a complete desktop version of the portfolio based on this [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1-1471)

You can see the live demonstration [here](https://lily4178993.github.io/Portfolio/).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -23,214 +23,242 @@
...
+                            A daily selection of privately personalized reads; no accounts
+                            or sign-ups required. has been the industry's standard.
+                        </p>
+                        <ul class="work-languages-list">
+                            <li class="work-language">HTML</li>
+                            <li class="work-language">Bootstrap</li>
+                            <li class="work-language">Ruby</li>
+                        </ul>
+                        <a href="#" class="btn see-work">See project</a>
+                    </div>
```
Comment (divinecharlotte) [inline index.html:116]: - [ ] kindly consider making sure that when the project is hovered on with the mouse the description and dark opacity disappears and a `See Project` button appears to the bottom of the image and takes the whole image width mentioned to meet the below requirement
> Lastly, when the project is hovered on with the mouse the description and dark opacity disappears and a `See Project` button appears to the bottom of the image and takes the whole image width mentioned button should take you to the Live Demo of the project or to the GitHub repository.

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
Hi @lily4178993    , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## To Highlight 💯 
- Descriptive Pr 👍
- linters are passing 👍 




 

## Required Changes ♻️

_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (wayungi) [reply index.html:116]: - [ ] The hover effect of the projects is not working yet. Please implement this feature.
hover

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @lily4178993 , 

image

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


 ## Highlights
- [ ] All linter checks are passing ✔️ 
- [ ]  Correct GitHub flow ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by IngDavidTM — APPROVED** [review]
Hi @lily4178993 🙋‍♂️, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**